default.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+  <remote name="origin" fetch="https://git.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git" />
+  <default revision="master" remote="origin"  sync-j="4" />
+  <project path="en/stacks/audio/alsa" name="alsa-utils" />
+  <project path="en/stacks/bluetooth" name="bluez" />
+  <project path="en/stacks/modem" name="modem-manager" />
+  <project path="en/stacks/network" name="network-manager" />
+  <project path="en/stacks/audio/pulseaudio" name="pulseaudio" />
+  <project path="en/stacks/tpm/tpm" name="tpm" />
+  <project path="en/stacks/tpm/tpm2" name="tpm2" />
+  <project path="en/stacks/disk/udisks" name="udisks" />
+  <project path="en/stacks/disk/udisks2" name="udisks2" />
```

Comment:
We only have a udisks2 snap. A udisks snap does not exist. If you're referring to https://code.launchpad.net/~snappy-hwe-team/snappy-hwe-snaps/+git/udisks then I don't know why that exists but it does not have any relevance and should be removed.